CMakeLists.txt

```diff
@@ -126,11 +126,13 @@ set(COGCORE_SOURCES
 )
 
 if (USE_SOUP2)
+    set(WPEWEBKIT_PC_NAME "wpe-webkit-1.0")
```

Comment:
AFAIK this assume using libsoup2 only with old versions of WebKit (<2.34) but ot is possible setup libsoup2 as well with the latest version of WebKit.

Reply:
Note that this is only the name of the `pkg-config` module. The suffix (`1.0` or `1.1`) is the **API** version—not the WPE WebKit version. When built with libsoup2 the API version of WPE WebKit is always `1.0`, and when built with libsoup3 it is always `1.1`.